src/main/java/com/github/princesslana/eriscasper/rest/InviteRoute.java

```diff
@@ -0,0 +1,38 @@
+package com.github.princesslana.eriscasper.rest;
+
+import com.github.princesslana.eriscasper.data.resource.Invite;
+import java.util.Optional;
+
+public class InviteRoute {
+
+  private final String path;
+
+  private InviteRoute(String code) {
+    this.path = "/invites/" + code;
+  }
+
+  /**
+   * @see <a href="https://discordapp.com/developers/docs/resources/invite#get-invite">
+   *     https://discordapp.com/developers/docs/resources/invite#get-invite</a>
+   */
+  public Route<Void, Invite> get(Optional<Boolean> withCounts) {
+    return Route.get(
+        path + withCounts.map(bool -> ";with_counts=" + bool).orElse(""), Invite.class);
```

Comment:
Is this mean to be a ? instead of ;

Reply:
Why not use QueryStringBuilder?

Reply:
Nope
https://stackoverflow.com/questions/39266970/what-is-the-difference-between-url-parameters-and-query-strings
https://discordapp.com/developers/docs/resources/invite#get-invite

Reply:
It's not a QueryString